src/Umbraco.Cms.Api.Management/Install/PostUnattendedInstallNotificationHandler.cs

```diff
@@ -12,27 +12,31 @@
 
 namespace Umbraco.Cms.Api.Management.Install;
 
-public class CreateUnattendedUserNotificationHandler : INotificationAsyncHandler<UnattendedInstallNotification>
+public class PostUnattendedInstallNotificationHandler : INotificationAsyncHandler<UnattendedInstallNotification>
```

Comment:
This is a public class, so unfortunately we can't rename in a minor version as strictly it's a binary breaking change.  I can see why you have renamed it though, as it does better reflect the behaviour.

In this case, given 17 is close, I'd suggest retargeting your PR to `v17/dev`.  We can then make this change to have a better named class.